docs/config.md

```diff
@@ -1,28 +1,20 @@
 # The go-ipfs config file
 
-The go-ipfs config file is a JSON document located at `$IPFS_PATH/config`. It
-is read once at node instantiation, either for an offline command, or when
-starting the daemon. Commands that execute on a running daemon do not read the
-config file at runtime.
+The go-ipfs config file is a JSON document located at `$IPFS_PATH/config`. It is read once at node instantiation, either for an offline command, or when starting the daemon. Commands that execute on a running daemon do not read the config file at runtime.
```

Comment:
Why reflow the text? Lots of users view these documents in text editors.

Reply:
Because the new files have no line breaks, just the old ones. So I removed all line breaks from all old files to have that consistent.

I don't know any text-editor which doesn't support soft-wrapping :)

Using no hard-wrapping has the advantage of using the available screen space completely.

Reply:
Let's just leave this. Mixing refactors like this with actual changes makes it impossible to review.

Reply:
Okay :)